WARN  2014-03-29 14:17:22,380 (NullSarosSession.java:64) unexpected method call
WARN  2014-03-29 14:17:22,395 (NullSarosSession.java:64) unexpected method call
WARN  2014-03-29 14:17:34,702 (NullSarosSession.java:64) unexpected method call
WARN  2014-03-29 14:17:34,718 (NullSarosSession.java:64) unexpected method call
WARN  2014-03-29 14:17:48,941 (StunServiceImpl.java:420) received no response from STUN server stunserver.org/132.177.123.6:3478 at local address /192.168.56.1:0
ERROR 2014-03-29 16:50:55,742 (SarosNet.java:95) XMPP connection error: 
java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at sun.security.ssl.InputRecord.readFully(Unknown Source)
	at sun.security.ssl.InputRecord.read(Unknown Source)
	at sun.security.ssl.SSLSocketImpl.readRecord(Unknown Source)
	at sun.security.ssl.SSLSocketImpl.readDataRecord(Unknown Source)
	at sun.security.ssl.AppInputStream.read(Unknown Source)
	at sun.nio.cs.StreamDecoder.readBytes(Unknown Source)
	at sun.nio.cs.StreamDecoder.implRead(Unknown Source)
	at sun.nio.cs.StreamDecoder.read(Unknown Source)
	at java.io.InputStreamReader.read(Unknown Source)
	at java.io.BufferedReader.fill(Unknown Source)
	at java.io.BufferedReader.read1(Unknown Source)
	at java.io.BufferedReader.read(Unknown Source)
	at org.xmlpull.mxp1.MXParser.fillBuf(MXParser.java:2992)
	at org.xmlpull.mxp1.MXParser.more(MXParser.java:3046)
	at org.xmlpull.mxp1.MXParser.nextImpl(MXParser.java:1144)
	at org.xmlpull.mxp1.MXParser.next(MXParser.java:1093)
	at org.jivesoftware.smack.PacketReader.parsePackets(PacketReader.java:316)
	at org.jivesoftware.smack.PacketReader.access$000(PacketReader.java:46)
	at org.jivesoftware.smack.PacketReader$1.run(PacketReader.java:72)
ERROR 2014-03-29 16:50:55,948 (SarosNet.java:385)  new connection state == ERROR
java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at sun.security.ssl.InputRecord.readFully(Unknown Source)
	at sun.security.ssl.InputRecord.read(Unknown Source)
	at sun.security.ssl.SSLSocketImpl.readRecord(Unknown Source)
	at sun.security.ssl.SSLSocketImpl.readDataRecord(Unknown Source)
	at sun.security.ssl.AppInputStream.read(Unknown Source)
	at sun.nio.cs.StreamDecoder.readBytes(Unknown Source)
	at sun.nio.cs.StreamDecoder.implRead(Unknown Source)
	at sun.nio.cs.StreamDecoder.read(Unknown Source)
	at java.io.InputStreamReader.read(Unknown Source)
	at java.io.BufferedReader.fill(Unknown Source)
	at java.io.BufferedReader.read1(Unknown Source)
	at java.io.BufferedReader.read(Unknown Source)
	at org.xmlpull.mxp1.MXParser.fillBuf(MXParser.java:2992)
	at org.xmlpull.mxp1.MXParser.more(MXParser.java:3046)
	at org.xmlpull.mxp1.MXParser.nextImpl(MXParser.java:1144)
	at org.xmlpull.mxp1.MXParser.next(MXParser.java:1093)
	at org.jivesoftware.smack.PacketReader.parsePackets(PacketReader.java:316)
	at org.jivesoftware.smack.PacketReader.access$000(PacketReader.java:46)
	at org.jivesoftware.smack.PacketReader$1.run(PacketReader.java:72)
WARN  2014-03-30 13:44:09,296 (StunServiceImpl.java:420) received no response from STUN server stunserver.org/132.177.123.6:3478 at local address /192.168.56.1:0
WARN  2014-03-30 13:44:30,734 (DiscoveryManager.java:445) Service Discovery failed on recipient gabrielwilde@gmail.com/MessagingAe01cc205 server: xmpp.l.google.com
No response from the server.: 
	at org.jivesoftware.smackx.ServiceDiscoveryManager.discoverInfo(ServiceDiscoveryManager.java:554)
	at org.jivesoftware.smackx.ServiceDiscoveryManager.discoverInfo(ServiceDiscoveryManager.java:512)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager.performServiceDiscovery(DiscoveryManager.java:442)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager.queryFeatureSupport(DiscoveryManager.java:379)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager.access$3(DiscoveryManager.java:346)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager$2.run(DiscoveryManager.java:325)
	at de.fu_berlin.inf.dpp.util.Utils$2.run(Utils.java:175)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
WARN  2014-03-30 13:45:00,750 (DiscoveryManager.java:445) Service Discovery failed on recipient gabrielwilde@gmail.com/MessagingAe01cc205 server: xmpp.l.google.com
No response from the server.: 
	at org.jivesoftware.smackx.ServiceDiscoveryManager.discoverInfo(ServiceDiscoveryManager.java:554)
	at org.jivesoftware.smackx.ServiceDiscoveryManager.discoverInfo(ServiceDiscoveryManager.java:512)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager.performServiceDiscovery(DiscoveryManager.java:442)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager.queryFeatureSupport(DiscoveryManager.java:379)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager.access$3(DiscoveryManager.java:346)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager$2.run(DiscoveryManager.java:325)
	at de.fu_berlin.inf.dpp.util.Utils$2.run(Utils.java:175)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
WARN  2014-03-30 14:05:15,602 (DiscoveryManager.java:445) Service Discovery failed on recipient gabrielwilde@gmail.com/MessagingAe01cc205 server: xmpp.l.google.com
No response from the server.: 
	at org.jivesoftware.smackx.ServiceDiscoveryManager.discoverInfo(ServiceDiscoveryManager.java:554)
	at org.jivesoftware.smackx.ServiceDiscoveryManager.discoverInfo(ServiceDiscoveryManager.java:512)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager.performServiceDiscovery(DiscoveryManager.java:442)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager.queryFeatureSupport(DiscoveryManager.java:379)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager.access$3(DiscoveryManager.java:346)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager$2.run(DiscoveryManager.java:325)
	at de.fu_berlin.inf.dpp.util.Utils$2.run(Utils.java:175)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
WARN  2014-03-30 14:05:45,612 (DiscoveryManager.java:445) Service Discovery failed on recipient gabrielwilde@gmail.com/MessagingAe01cc205 server: xmpp.l.google.com
No response from the server.: 
	at org.jivesoftware.smackx.ServiceDiscoveryManager.discoverInfo(ServiceDiscoveryManager.java:554)
	at org.jivesoftware.smackx.ServiceDiscoveryManager.discoverInfo(ServiceDiscoveryManager.java:512)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager.performServiceDiscovery(DiscoveryManager.java:442)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager.queryFeatureSupport(DiscoveryManager.java:379)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager.access$3(DiscoveryManager.java:346)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager$2.run(DiscoveryManager.java:325)
	at de.fu_berlin.inf.dpp.util.Utils$2.run(Utils.java:175)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
ERROR 2014-03-30 15:55:41,667 (SarosNet.java:95) XMPP connection error: 
java.net.SocketTimeoutException: Read timed out
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at sun.security.ssl.InputRecord.readFully(Unknown Source)
	at sun.security.ssl.InputRecord.read(Unknown Source)
	at sun.security.ssl.SSLSocketImpl.readRecord(Unknown Source)
	at sun.security.ssl.SSLSocketImpl.readDataRecord(Unknown Source)
	at sun.security.ssl.AppInputStream.read(Unknown Source)
	at sun.nio.cs.StreamDecoder.readBytes(Unknown Source)
	at sun.nio.cs.StreamDecoder.implRead(Unknown Source)
	at sun.nio.cs.StreamDecoder.read(Unknown Source)
	at java.io.InputStreamReader.read(Unknown Source)
	at java.io.BufferedReader.fill(Unknown Source)
	at java.io.BufferedReader.read1(Unknown Source)
	at java.io.BufferedReader.read(Unknown Source)
	at org.xmlpull.mxp1.MXParser.fillBuf(MXParser.java:2992)
	at org.xmlpull.mxp1.MXParser.more(MXParser.java:3046)
	at org.xmlpull.mxp1.MXParser.nextImpl(MXParser.java:1144)
	at org.xmlpull.mxp1.MXParser.next(MXParser.java:1093)
	at org.jivesoftware.smack.PacketReader.parsePackets(PacketReader.java:316)
	at org.jivesoftware.smack.PacketReader.access$000(PacketReader.java:46)
	at org.jivesoftware.smack.PacketReader$1.run(PacketReader.java:72)
ERROR 2014-03-30 15:55:41,917 (SarosNet.java:385)  new connection state == ERROR
java.net.SocketTimeoutException: Read timed out
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at sun.security.ssl.InputRecord.readFully(Unknown Source)
	at sun.security.ssl.InputRecord.read(Unknown Source)
	at sun.security.ssl.SSLSocketImpl.readRecord(Unknown Source)
	at sun.security.ssl.SSLSocketImpl.readDataRecord(Unknown Source)
	at sun.security.ssl.AppInputStream.read(Unknown Source)
	at sun.nio.cs.StreamDecoder.readBytes(Unknown Source)
	at sun.nio.cs.StreamDecoder.implRead(Unknown Source)
	at sun.nio.cs.StreamDecoder.read(Unknown Source)
	at java.io.InputStreamReader.read(Unknown Source)
	at java.io.BufferedReader.fill(Unknown Source)
	at java.io.BufferedReader.read1(Unknown Source)
	at java.io.BufferedReader.read(Unknown Source)
	at org.xmlpull.mxp1.MXParser.fillBuf(MXParser.java:2992)
	at org.xmlpull.mxp1.MXParser.more(MXParser.java:3046)
	at org.xmlpull.mxp1.MXParser.nextImpl(MXParser.java:1144)
	at org.xmlpull.mxp1.MXParser.next(MXParser.java:1093)
	at org.jivesoftware.smack.PacketReader.parsePackets(PacketReader.java:316)
	at org.jivesoftware.smack.PacketReader.access$000(PacketReader.java:46)
	at org.jivesoftware.smack.PacketReader$1.run(PacketReader.java:72)
WARN  2014-03-31 17:13:49,343 (StunServiceImpl.java:420) received no response from STUN server stunserver.org/132.177.123.6:3478 at local address /192.168.56.1:0
WARN  2014-03-31 17:13:49,343 (StunServiceImpl.java:420) received no response from STUN server stunserver.org/132.177.123.6:3478 at local address /141.22.64.13:0
ERROR 2014-03-31 17:13:59,910 (SarosNet.java:385)  new connection state == ERROR
googlemail.com:5222 Exception: XMPPError connecting to googlemail.com:5222.; : remote-server-error(502)
	at org.jivesoftware.smack.XMPPConnection.connectUsingConfiguration(XMPPConnection.java:600)
	at org.jivesoftware.smack.XMPPConnection.connect(XMPPConnection.java:1022)
	at de.fu_berlin.inf.dpp.net.SarosNet.connect(SarosNet.java:250)
	at de.fu_berlin.inf.dpp.Saros.connect(Saros.java:660)
	at de.fu_berlin.inf.dpp.ui.actions.ChangeXMPPAccountAction$6.run(ChangeXMPPAccountAction.java:195)
	at de.fu_berlin.inf.dpp.util.Utils$2.run(Utils.java:175)
	at java.lang.Thread.run(Unknown Source)
WARN  2014-03-31 17:14:10,726 (StunServiceImpl.java:420) received no response from STUN server stunserver.org/132.177.123.6:3478 at local address /141.22.64.13:0
WARN  2014-03-31 17:14:10,726 (StunServiceImpl.java:420) received no response from STUN server stunserver.org/132.177.123.6:3478 at local address /192.168.56.1:0
WARN  2014-03-31 17:14:19,292 (NullSarosSession.java:64) unexpected method call
WARN  2014-03-31 17:14:19,308 (NullSarosSession.java:64) unexpected method call
ERROR 2014-03-31 17:14:21,370 (SarosNet.java:385)  new connection state == ERROR
googlemail.com:5222 Exception: XMPPError connecting to googlemail.com:5222.; : remote-server-error(502)
	at org.jivesoftware.smack.XMPPConnection.connectUsingConfiguration(XMPPConnection.java:600)
	at org.jivesoftware.smack.XMPPConnection.connect(XMPPConnection.java:1022)
	at de.fu_berlin.inf.dpp.net.SarosNet.connect(SarosNet.java:250)
	at de.fu_berlin.inf.dpp.Saros.connect(Saros.java:660)
	at de.fu_berlin.inf.dpp.ui.actions.ChangeXMPPAccountAction$6.run(ChangeXMPPAccountAction.java:195)
	at de.fu_berlin.inf.dpp.util.Utils$2.run(Utils.java:175)
	at java.lang.Thread.run(Unknown Source)
WARN  2014-03-31 17:14:21,370 (SWTUtils.java:144) could not execute runnable de.fu_berlin.inf.dpp.ui.widgets.ConnectionStateComposite$1$1@2608e8d, UI thread is not available
StackTrace:
	at de.fu_berlin.inf.dpp.ui.util.SWTUtils.runSafeSWTAsync(SWTUtils.java:145)
	at de.fu_berlin.inf.dpp.ui.widgets.ConnectionStateComposite$1.connectionStateChanged(ConnectionStateComposite.java:55)
	at de.fu_berlin.inf.dpp.net.SarosNet.setConnectionState(SarosNet.java:390)
	at de.fu_berlin.inf.dpp.net.SarosNet.connect(SarosNet.java:273)
	at de.fu_berlin.inf.dpp.Saros.connect(Saros.java:660)
	at de.fu_berlin.inf.dpp.ui.actions.ChangeXMPPAccountAction$6.run(ChangeXMPPAccountAction.java:195)
	at de.fu_berlin.inf.dpp.util.Utils$2.run(Utils.java:175)
	at java.lang.Thread.run(Unknown Source)
WARN  2014-03-31 17:14:21,386 (SWTUtils.java:144) could not execute runnable de.fu_berlin.inf.dpp.ui.widgets.ConnectionStateComposite$1$1@1b993911, UI thread is not available
StackTrace:
	at de.fu_berlin.inf.dpp.ui.util.SWTUtils.runSafeSWTAsync(SWTUtils.java:145)
	at de.fu_berlin.inf.dpp.ui.widgets.ConnectionStateComposite$1.connectionStateChanged(ConnectionStateComposite.java:55)
	at de.fu_berlin.inf.dpp.net.SarosNet.setConnectionState(SarosNet.java:390)
	at de.fu_berlin.inf.dpp.net.SarosNet.connect(SarosNet.java:275)
	at de.fu_berlin.inf.dpp.Saros.connect(Saros.java:660)
	at de.fu_berlin.inf.dpp.ui.actions.ChangeXMPPAccountAction$6.run(ChangeXMPPAccountAction.java:195)
	at de.fu_berlin.inf.dpp.util.Utils$2.run(Utils.java:175)
	at java.lang.Thread.run(Unknown Source)
WARN  2014-03-31 17:14:21,402 (SWTUtils.java:193) could not execute runnable de.fu_berlin.inf.dpp.ui.util.SWTUtils$1@1a507b20, UI thread is not available
StackTrace:
	at de.fu_berlin.inf.dpp.ui.util.SWTUtils.runSafeSWTSync(SWTUtils.java:194)
	at de.fu_berlin.inf.dpp.ui.util.SWTUtils.runSWTSync(SWTUtils.java:157)
	at de.fu_berlin.inf.dpp.ui.util.DialogUtils.popUpYesNoQuestion(DialogUtils.java:242)
	at de.fu_berlin.inf.dpp.Saros.connect(Saros.java:675)
	at de.fu_berlin.inf.dpp.ui.actions.ChangeXMPPAccountAction$6.run(ChangeXMPPAccountAction.java:195)
	at de.fu_berlin.inf.dpp.util.Utils$2.run(Utils.java:175)
	at java.lang.Thread.run(Unknown Source)
ERROR 2014-03-31 17:14:21,417 (DialogUtils.java:250) An internal error ocurred while trying to open the question dialog.
WARN  2014-03-31 17:14:21,948 (SWTUtils.java:144) could not execute runnable de.fu_berlin.inf.dpp.ui.widgets.ConnectionStateComposite$1$1@373fd1d8, UI thread is not available
StackTrace:
	at de.fu_berlin.inf.dpp.ui.util.SWTUtils.runSafeSWTAsync(SWTUtils.java:145)
	at de.fu_berlin.inf.dpp.ui.widgets.ConnectionStateComposite$1.connectionStateChanged(ConnectionStateComposite.java:55)
	at de.fu_berlin.inf.dpp.net.SarosNet.setConnectionState(SarosNet.java:390)
	at de.fu_berlin.inf.dpp.net.SarosNet.connect(SarosNet.java:248)
	at de.fu_berlin.inf.dpp.Saros.connect(Saros.java:660)
	at de.fu_berlin.inf.dpp.ui.actions.ChangeXMPPAccountAction$6.run(ChangeXMPPAccountAction.java:195)
	at de.fu_berlin.inf.dpp.util.Utils$2.run(Utils.java:175)
	at java.lang.Thread.run(Unknown Source)
